src/main/java/org/cascadebot/cascadebot/commands/management/SettingsCommand.java

```diff
@@ -53,11 +47,12 @@ public void onCommand(Member sender, CommandContext context) {
                 try {
                     Setting settingAnnotation = field.getAnnotation(Setting.class);
                     if (settingAnnotation != null) {
-                        if (!context.getData().getEnabledFlags().containsAll(Arrays.asList(settingAnnotation.flagRequired()))) {
+                        /*if (!context.getData().getEnabledFlags().containsAll(Arrays.asList(settingAnnotation.flagRequired()))) {
                             String niceName = Language.i18n(context.getGuild().getIdLong(), "settings." + field.getDeclaringClass().getAnnotation(SettingsContainer.class).module().name().toLowerCase() + "." + field.getName() + ".nice_name");
                             context.getTypedMessaging().replyDanger(context.i18n("commands.settings.cannot_edit", niceName));
                             return;
                         }
+                         */
```

Comment:
Why has the new type of tier not been added to this?

Reply:
I didn't know settings had flags to them